scripts/rmc.py

```diff
@@ -31,6 +48,17 @@ def delete_file(filename):
     except OSError:
         pass
 
+# Add a set of default CBMC arguments
+def add_default_cbmc_flags(args):
+    if not args.no_default_flags:
```

Comment:
Is there a reason we block on `no_default_flags` both here and in `rmc`/`cargo-rmc`?

Reply:
It's a mistake, thanks for pointing this out. The condition is always checked before calling this function.